addons/hr_expense/tests/test_expenses.py

```diff
@@ -1753,3 +1753,28 @@ def test_expense_product_update(self):
             {'name': 'test sheet no update', 'unit_amount': 100.0, 'quantity': 1, 'total_amount': 100.0},
             {'name':    'test sheet update', 'unit_amount': 250.0, 'quantity': 1, 'total_amount': 250.0},  # no update
         ])
+
+    def test_foreign_currencies_total(self):
+        foreign_currency = self.env['res.currency'].create({
+            'name': 'DBL',
+            'symbol': 'DD',
+            'rounding': 0.01,
+            'currency_unit_label': 'Doubloon'
+        })
```

Comment:
You should be able to use `self.currency_data['currency']` since it is a subclass of `AccountTestInvoicingCommon`